src/styles/blocks/stars.css

```diff
@@ -0,0 +1,37 @@
+.stars {
+  display: flex;
+}
+
+.stars--1 :nth-child(-n + 1) {
```

Comment:
Remove the space between `.stars--1` and `:nth-child(-n + 1)` to correctly target the first child elements directly inside `.stars--1`.